spinalcordtoolbox/data/csa_normalization_models/coeff_brain_thalamus_sex.csv

```diff
@@ -0,0 +1,6 @@
+,coeff,mean
+const,33.083923849329736,
```

Comment:
What is `const`, and why do we include it here if we're just going to drop it in SCT?

Reply:
the models are linear regressions, so `const `is the constant of the equation. We don't need it here, I'll tweak the code in https://github.com/sct-pipeline/ukbiobank-spinalcord-csa so the .csv file don't include it anymore